docker-start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p /data/db
+mongod &
+mkdir instance
+echo 'DEBUG=False' >> instance/config.py
+echo "SECRET_KEY='?\xbf,\xb4\x8d\xa3<\x9c\xb0@\x0f5\xab,w\xee\x8d$0\x13\x8b83'" >> instance/config.py
```

Comment:
I'm not exactly fond of setting "a" secret key there, but I guess it's ok for the sake of development.

Should we aim to have this docker image eventually be good for deployment though?

Reply:
I could generate a random key every time this script is run ?

Reply:
Yes, hardcoded production secret key - not so good. Randomly creating a new one on every start isn't probably a good idea either. The key is used to sign user sessions, so in case of a restart users would lose their ongoing sessions. Although this might be a corner case, I think we can find a better solution. Just search the web for how others create Flask app docker files.

Reply:
Couldn't find anything regarding secret keys for dockerized flask app.